libs/docs/core/grid-list/examples/index.ts

```diff
@@ -29,5 +30,6 @@ export const COMPONENTS = [
     GridListDndExampleComponent,
     GridListLayoutExampleComponent,
     GridListFocusingItemExampleComponent,
-    GridListAutoHeightExampleComponent
+    GridListAutoHeightExampleComponent,
```

Comment:
You need to remove the imports of the components that are not used as well as their registration.